tests/test_util.py

```diff
@@ -74,3 +86,68 @@ def test_zipsafe():
     for zip_safe in (False, True):
       with make_egg(zipped=zipped, zip_safe=zip_safe) as dist:
         assert DistributionHelper.zipsafe(dist) is zip_safe
+
+
+@mock.patch('pex.util.resource_string', spec=pkg_resources.resource_string)
+@mock.patch('pex.util.resource_isdir', spec=pkg_resources.resource_isdir)
+@mock.patch('pex.util.resource_listdir', spec=pkg_resources.resource_listdir)
+def test_access_zipped_assets(mock_resource_listdir, mock_resource_isdir, mock_resource_string):
+  try:
+    import __builtin__
+    builtin_path = __builtin__.__name__
+  except ImportError:
+    # Python3
+    import builtins
+    builtin_path = builtins.__name__
+
+  mock_open = mock.mock_open()
+  with mock.patch('%s.open' % builtin_path, mock_open, create=True):
```

Comment:
We're already using mock_open, but need to patch it in still.

Hm, I don't think so:

```
[tw-mbp-jsmith pex (jsmith/access_zipped_assets)]$ python
Python 2.7.9 (default, Mar  4 2015, 17:22:02) 
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.42)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import builtins
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named builtins
>>> 
```